zenoh-protocol/tests/channel.rs

```diff
@@ -326,3 +347,76 @@ fn channel_tcp_udp() {
         close_session(router_manager, client_session, locators).await;
     });
 }
+
+#[cfg(all(feature = "unix", target_family = "unix"))]
+#[test]
+fn channel_tcp_unix() {
+    let _ = std::fs::remove_file("/tmp/socket_1.sock");
```

Comment:
Although /tmp is available in the totality of the system, I believe that using the current directory for the unix socket path is a better approach:

It is guaranteed that the cargo test will have write permissions while executing the test
System files are not created outside the test directory
I also suggest to use a more explanatory name for the socket, e.g., zenoh-test-unix-socket_1.sock

Reply:
Another solution is use the [tempfile](https://crates.io/crates/tempfile) that relies on [std::env::temp_dir](https://doc.rust-lang.org/std/env/fn.temp_dir.html) and performs automatic deletion of file.

Reply:
I like the idea of using the [tempfile](https://crates.io/crates/tempfile) crate. This should be added to the dev dependencies.